From e1b3b76a8625b792129bdcbb2dee1f7b6b85e700 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:50:03 -0400
Subject: [tools] perf/archive: Remove duplicated 'runs' in man page

Message-id: <1408953210-26343-116-git-send-email-jolsa@redhat.com>
Patchwork-id: 88834
O-Subject: [PATCH RHEL7.1 BZ1133083 115/322] perf archive: Remove duplicated 'runs' in man page
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit f113bee01956d4a2efde461443b09ab1020f4b17
Author: Dongsheng Yang <yangds.fnst@cn.fujitsu.com>
Date: Mon Dec 9 17:47:47 2013 -0500

description
===========
Two 'runs' here breaks the sentence in Description of 'perf archive'
command.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/Documentation/perf-archive.txt b/tools/perf/Documentation/perf-archive.txt
index 5032a14..ac6ecbb 100644
--- a/tools/perf/Documentation/perf-archive.txt
+++ b/tools/perf/Documentation/perf-archive.txt
@@ -12,9 +12,9 @@ SYNOPSIS
 
 DESCRIPTION
 -----------
-This command runs runs perf-buildid-list --with-hits, and collects the files
-with the buildids found so that analysis of perf.data contents can be possible
-on another machine.
+This command runs perf-buildid-list --with-hits, and collects the files with the
+buildids found so that analysis of perf.data contents can be possible on another
+machine.
 
 
 SEE ALSO
-- 
1.7.1